core/datainput.gms

```diff
@@ -394,6 +395,11 @@ $endif.floorscen
 
 *** In case regionally differentiated investment costs should be used the corresponding entries are revised:
 $ifthen.REG_techcosts not "%cm_techcosts%" == "GLO"   !! cm_techcosts is REG or REG2040
+*** calculate regional floor costs for learning technologies from ratio of global values
+*** take the ratio of the global floorcost to global initial cost, and multiply with the new regional cost to get new floorcost that should create reasonable learning paths around 2020
+    pm_data(regi,"floorcost",teLearn(te))$(teRegTechCosts(te) ) = p_inco0("2015",regi,te) * fm_dataglob("floorcost",te) / fm_dataglob("inco0",te);
+    pm_data(regi,"floorcost","spv") = p_inco0("2020",regi,"spv") * fm_dataglob("floorcost","spv") / fm_dataglob("inco0","spv") ; !! 2020 values are available for PV
+
```

Comment:
if I am not mistaken, this block essentially is supposed to replace $ifthen.floorscen %cm_floorCostScen% == "pricestruc" above https://github.com/remindmodel/remind/pull/2129/files#diff-328de481dd31565bdd16ff7586e7d89a354d406b3a43cdde4df0f6613e9439d1R371-R383

which was anyhow wrong - as we were talking about before, because it was pegged to the maximum

however, take care that there are historical data newer 2015, in which case 2020 should be used, see L377-379